packages/data-transport-layer/src/utils/common.ts

```diff
@@ -1,16 +1,3 @@
-import { toHexString } from '@eth-optimism/core-utils'
-
-/**
- * Basic timeout-based async sleep function.
- *
- * @param ms Number of milliseconds to sleep.
- */
-export const sleep = async (ms: number): Promise<void> => {
-  return new Promise<void>((resolve) => {
-    setTimeout(resolve, ms)
-  })
-}
-
 export const assert = (condition: () => boolean, reason?: string) => {
```

Comment:
I think you can also delete this function, which means you should be able to delete the entire file

Reply:
You are right, I didn't notice `assert ` function is not used in that package.